app/src/main/java/com/yeonae/chamelezone/util/LocationDistance.kt

```diff
@@ -19,5 +20,5 @@ fun distanceByDegree(
 
     val distance = startPos.distanceTo(endPos) / 1000
 
-    return "$distance km"
+    return "${(distance * 100).roundToInt() /100.0} km"
```

Comment:
```suggestion
    return "${(distance * 100).roundToInt() / 100.0f} km"
```
간단히 소수점 1자리를 보여주는 것이라면 double type보다는 float type의 사용이 적절하지 않을까 싶고, (byte 크기사용이 float이 더 적으므로)
곱하고 나누는 연산이 두번 들어가는 것이기 때문에
```suggestion
    return String.format("%.1f", distance) 
```
이런 사용은 어떨까 싶네요. (반올림 됩니다.)

Reply:
넵 수정했습니다!
e4187a3